### PR TITLE
Homepage refresh

### PR DESCRIPTION
**Pull request by dannypernik**

### CODE REVIEW

File: log/views.log [hunk 1]
```diff
@@ -0,0 +1,205 @@
+views.logERROR:openCurrents.views:Invalid signup request: {'user_lastname': [ValidationError([u'This field is required.'])], 'user_firstname': [ValidationError([u'This field is required.'])], 'user_email': [ValidationError([u'This field is required.'])]}
```
Comment (nickolashe) [inline log/views.log]: Remove this file from your this PR using:
git rm --cached log/views.log
git commit -m "remove log"

Then push with:
git push origin danny

File: openCurrents/templates/view-project.ejs [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
```
Comment (nickolashe) [inline openCurrents/templates/view-project.ejs]: What is this file?

Comment (ronakulus) [inline openCurrents/templates/view-project.ejs]: I looked at the text and remember @dannypernik mentioning he wanted to get a place in Austin as a headquarters for oC but I am just speculating.

Comment (dannypernik) [inline openCurrents/templates/view-project.ejs]: This file is from when I was working with Ruby. I've deleted it.

**Pull request closed without merging**